orion/sites/admin/src/index.js

```diff
@@ -18,8 +18,13 @@ import { AnnouncePage } from './announce/announce.js';
 import { AnnounceEditPage } from './announce/announceEdit.js';
 import { LocationPage } from './location/location.js';
 import { LocationEditPage } from './location/locationEdit.js';
+<<<<<<< HEAD
+import { SemesterPage } from './semester/semester.js';
+import { SemesterEditPage } from './semester/semesterEdit.js';
+=======
 import { SemesterPage } from './semesters/semester.js';
 import { SemesterEditPage } from './semesters/semesterEdit.js';
+>>>>>>> 69085cf1cf077c270ee68280d22d79089a85928a
```

Comment:
I think your folder is called semester. So we want to keep `./semester/semester.js` but remove the one with folders named `semesters`